test_scripts/WebEngine/commonWebEngine.lua

```diff
@@ -321,6 +508,83 @@ function common.checkUpdateAppList(pPolicyAppID, pTimes, pExpNumOfApps)
   common.wait()
 end
 
+--[[ @getOnSystemCapabilityParams: Provide default parameters for 'OnSystemCapabilityParams' RPC
```

Comment:
The function description can be removed because all other functions do not contain a description.